src/static/js/modules/prepare-worksheets/inputs/button-grading-group.js

```diff
@@ -11,42 +11,39 @@ function ButtonGradingGroup(options) {
   // Note bind()'s lack of IE8 support.
   var _self = this;
 
-  var _grades = require('./input-graded-grades');
-
-  // The grade of this input.
-  var grade = _grades.UNSET;
-  var lastNode;
-
+  this.row = options.row;
```

Comment:
Additional to the above, I'd suggest creating a locally scoped var and then doing all the exposing of the methods/properties at the end. So like this:

```javascript
var row = options.row;

...

  // Expose public methods
  this.getGrade = getGrade;
  this.setGrade = setGrade;
  this.row = row;
```

You may have to create a method instead of exposing the property directly, to avoid problems:

```javascript
var _row = options.row;

...
  // @return [Object] A row of data.
  function getRow() {
    return _row;
  }

  // Expose public methods
  this.getGrade = getGrade;
  this.setGrade = setGrade;
  this.getRow = getRow;
```

**Note:** I use the underscore prefix to designate what is not being exposed externally.